config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: { registrations: 'registrations' }
+  resources :tweets
```

Comment:
It creates the application routes using resources :tweets, only: [:new, :create, :index]